tests/unit/routes/test_patient_router.py

```diff
@@ -210,7 +210,7 @@ def test_get_orphaned_patients_with_limit(self, client):
             },
         }
 
-        response = client.get("/patient/orphaned?limit=2")
+        response = client.get("/api/patient/orphaned?limit=2")
         assert response.json() == {
```

Comment:
For some reasons the assertions in this file are not getting updated with the `/api` prefix, an example of this is line 218. This also happened for file `test_person_router.py` where all assertions pass when the `/api` is not present.